src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -309,7 +309,7 @@ class LicenceController(
   fun getRecentlyApprovedLicences(
```

Comment:
now returning for getRecentlyApprovedLicences - this technically isn't required but is conceptually simpler: using the same bespoke representation for the approver in both tabs.